lib/OpenQA/BuildResults.pm

```diff
@@ -181,7 +181,7 @@ sub compute_build_results ($group, $limit, $time_limit_days, $tags, $subgroup_fi
     my $max_jobs = 0;
     my $now = DateTime->now;
     for my $build (@builds) {
-        last if defined($limit) && (--$limit < 0);
+        #last if defined($limit) && (--$limit < 0);
```

Comment:
This is normally limited to 3 or 10 depending on the page. I'm wondering on the performance impact of running this loop on all the builds on a production database like the OSD database. I can imagine that it becomes notably slower, especially on the index page.

Reply:
yea I guess that might be a concern. My thought was that the default 400 limit is not that bad.

Reply:
btw this line can stay and return immediately if $limit == 0